packages/login/src/components/PasswordRequest/PasswordRequest.js

```diff
@@ -32,25 +36,29 @@ export class PasswordRequest extends React.Component {
 
   render() {
     return (
-      <StyledPasswordRequest className="login-form">
+      <StyledLoginFormWrapper>
```

Comment:
Why append "Wrapper" in this name? Usually you just call is Styled{Component}?

Reply:
The naming convention Styled{Component} does only work if each component has one styled component associated. Purpose of <StyledLoginFormWrapper> is to stretch and center <StyledLoginForm>. Therefore it is a wrapper. In means of clarity, versatility and reusability it makes sense to split styling into several chunks. May it make sense to use more generic names and to place the commonly used styled components alltogether and only in StyledLogin.js? Currently following files contain styled components.
./LoginForm/StyledLoginForm.js
./StyledLoginForm.js
./TwoStepLoginForm/StyledTwoStepLoginForm.js
./passwordUpdate/PasswordUpdateDialog/StyledPasswordInput.js
./passwordUpdate/PasswordUpdateDialog/StyledPasswordUpdateDialog.js

Reply:
The sole purpose of `StyledLoginFormWrapper` is to center (and stretch)? IMO it would be a lot clearer if this component is called something with center/stretch.

Reply:
Stretch is only one of several purposes. A concise name is not available. I established a generic yet a homogen naming for fields.